### ISSUE TITLE
Ability to exclude Default VPC

### ISSUE THREAD

**Issue by leozhad** [rebuy-de/aws-nuke#188]
I would love to be able to exclude the default VPC via the config file

**Comment by svenwltr** [comment]
Will be implemented in #198. The filter should look like this:

```yaml
accounts:
  "1234567890":
    filters:
      EC2VPC:
      - property: IsDefault
        value: "true"
```

**Comment by svenwltr** [comment]
This should be fixed in [v2.1.0-beta.1](https://github.com/rebuy-de/aws-nuke/releases/tag/v2.1.0-beta.1).
